apps/judicial-system/web/src/routes/Court/Indictments/Overview/Overview.tsx

```diff
@@ -2,7 +2,7 @@ import { useCallback, useContext, useState } from 'react'
 import { useIntl } from 'react-intl'
```

Comment:
There is nothing new here except the data button. But I started a refactor to extract the child overview body component since I thought originally I would need to add some if conditions for those components which was not needed in the end, but decided to keep the `OverviewBody` component since it simplifies a bit the parent component.